### ISSUE TITLE
Time off

### ISSUE THREAD

**Issue by BhupendrakVirim** [odoo/odoo#146101]
Impacted versions: 17

Steps to reproduce: create leave 

Current behavior: show warning in default leave like paid time off and with new leave type (There is no valid allocation to cover that request.)

Expected behavior: leave create without any warning

Video/Screenshot link (optional): 
![image](https://github.com/odoo/odoo/assets/92737875/e2ca57ac-110d-49ff-bb78-8458240986a8)

Support ticket number submitted via odoo.com/help (optional):

**Comment by virimudbhav** [comment]
Yes mate, I am also getting the same issue. Seems to be working fine on odoo runbot though!

**Comment by MuhamedAbdalla** [comment]
@vava-odoo 
I'm willing to work on this issue.
Can I get the sign to start trying, Please?

**Comment by MuhamedAbdalla** [comment]
@vava-odoo 
Can I check this, Please?

**Comment by techgpc** [comment]
@BhupendrakVirim Did you found solution for this

**Comment by SLiesk** [comment]
Push
